functions/api/product.js

```diff
@@ -426,48 +319,19 @@ function validateParams(body, type) {
                 thresholds: joi.object()
             })
             break
-        case constants.STATUS:
-            schema = joi.object({
-                product: joi.string()
-                    .min(1)
-                    .max(30)
-                    .required(),
-                isActive: joi.bool()
-                    .required()
-            })
-            break
-        case constants.CATEGORY:
+        case constants.UPDATE:
             schema = joi.object({
                 product: joi.string()
                     .min(1)
-                    .max(30)
-                    .required(),
+                    .max(30),
                 category: joi.string()
                     .min(1)
-                    .max(30)
-                    .required()
-            })
-            break
-        case constants.UNIT:
-            schema = joi.object({
-                product: joi.string()
-                    .min(1)
-                    .max(30)
-                    .required(),
+                    .max(30),
                 unit: joi.string()
                     .min(1)
-                    .max(30)
-                    .required()
-            })
-            break
-        case constants.THRESHOLD:
-            schema = joi.object({
-                product: joi.string()
-                    .min(1)
-                    .max(30)
-                    .required(),
-                thresholds: joi.object()
-                    .required()
+                    .max(30),
+                thresholds: joi.object(),
```

Comment:
u can add one more level of validation: inside this object what you are expecting

Reply:
Thresholds object will have n number for key value pairs. how to add checks for joi.anyString():value?

Reply:
"thresholds": {
		"kormangala": 15
	}